stevedore-build-artifacts.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+artifact_path='producedbuilds/*'
```

Comment:
This appears unused and should probably be removed.

Reply:
done